src/src/AuthService.js

```diff
@@ -27,12 +27,22 @@ const msalInstance = new PublicClientApplication({
 
 export { msalInstance as MsalInstance };
 
-export function callApi(url, accessToken, method = "GET", payload = null) {
+export function callApi(
+  url,
+  accessToken,
+  method = "GET",
+  payload = null,
+  falseUserPermissions = false,
+) {
   const headers = new Headers();
 
   const bearer = `Bearer ${accessToken}`;
   headers.append("Authorization", bearer);
 
+  if (!falseUserPermissions) {
```

Comment:
`if we do not have false permissions, send permissions=1`
This feels hard to read, and I am quite certain we can clean it up without too much hassle :)